wrappers/_shared.nix

```diff
@@ -23,21 +23,35 @@ let
     map
     mkIf
     mkMerge
-    mkOption
     optionalAttrs
     setAttrByPath
     ;
   cfg = config.programs.nixvim;
+
+  nixpkgsModule =
+    { config, ... }:
+    {
+      _file = ./_shared.nix;
+
+      config = {
+        nixpkgs = {
+          # Use global packages by default in nixvim's submodule
+          # FIXME: probably doesn't need to be mkForce
+          # What should happen if the user configures both `nixpkgs.pkgs` and `nixpkgs.useGlobalPackages` ?
```

Comment:
I think that we should either raise a warning and ignore one of them (I don't know which ...) or raise an error